src/libs/ReportUtils.ts

```diff
@@ -2176,9 +2171,12 @@ function getDeletedParentActionMessageForChatReport(reportAction: OnyxEntry<Repo
  * Returns the preview message for `REIMBURSEMENT_QUEUED` action
  */
 function getReimbursementQueuedActionMessage(reportAction: OnyxEntry<ReportAction>, reportOrID: OnyxEntry<Report> | string, shouldUseShortDisplayName = true): string {
+    if (!ReportActionsUtils.isMoneyRequestAction(reportAction)) {
+        return '';
+    }
```

Comment:
This check seems to be causing this deploy blocker: https://github.com/Expensify/App/issues/44338

The actionName is `"REIMBURSEMENTQUEUED"`

Reply:
Fixing here: https://github.com/Expensify/App/pull/44357